android/src/main/java/com/snowplowanalytics/react/tracker/RNSnowplowTrackerModule.java

```diff
@@ -73,6 +76,26 @@ public void createTracker(ReadableMap argmap,
             } else {
                 networkConfiguration = new NetworkConfiguration(networkConfig.getString("endpoint"));
             }
+            if (networkConfig.hasKey("requestHeaders") && !networkConfig.isNull("requestHeaders")) {
+                ReadableMap requestHeaders = networkConfig.getMap("requestHeaders");
+                if (requestHeaders != null) {
+                    OkHttpClient client = new OkHttpClient.Builder()
```

Comment:
this is nice workaround! still, it will be great when the Android tracker exposes this option as well..